python/console/console.py

```diff
@@ -96,6 +140,33 @@ def closeEvent(self, event):
         QWidget.closeEvent(self, event)
 
 
+class PythonConsoleW(QWidget):
+    """ as a QWidget """
```

Comment:
Astyle also complains about missing empty line between the comment and the method definition.